src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -43,6 +44,7 @@ const SearchPage = () => {
       setResults(posts.values);
     } catch (error) {
       console.error('Something went wrong while fetching data', error);
+      setSearchError(true);
```

Comment:
Set it to the `error` object vs. `true` so we have access to it, in case we ever need to use data within it.